app/gui/src/ide.rs

```diff
@@ -104,8 +103,6 @@ pub struct FailedIde {
 
 
 /// The Path of the module initially opened after opening project in IDE.
-pub fn initial_module_path(project: &model::Project) -> FallibleResult<model::module::Path> {
-    model::module::Path::from_name_segments(project.project_content_root_id(), &[
-        INITIAL_MODULE_NAME,
-    ])
+pub async fn initial_module_path(project: &model::Project) -> FallibleResult<model::module::Path> {
+    Ok(project.main_module_model().await?.path().clone_ref())
```

Comment:
Getting a module from the project is a quite big deal: That may mean opening the module in the engine, and setting up the edit updates. That's why main_module_model is asynchronous. I think we don't want to do run the entire process only to get the path of the module.

Reply:
Fixed